app/mobile/lib/presentation/states/game_list_grid_view_state.dart

```diff
@@ -0,0 +1,114 @@
+          name: "Call of Duty: WWII",
+          imageLink:
+              "https://upload.wikimedia.org/wikipedia/tr/8/85/Call_of_Duty_WIII_Kapak_Resmi.jpg"),
+      Game(
+          id: 4,
+          description:
+              "Celeste, Kanadalı video oyunu geliştiricisi Matt Thorson tarafından tasarlanan ve geliştirilen bir platform oyunudur. Noel Berry tarafından tasarlanan ve geliştirilen bir platform oyunu olan TowerFall'ın yaratıcısı Matt Thorson tarafından tasarlanan ve geliştirilen bir platform oyunudur. Oyun, 2018'de Microsoft Windows, Nintendo Switch, PlayStation 4, Xbox One, macOS ve Linux için piyasaya sürüldü. Oyun, 2019'da Google Stadia için piyasaya sürüldü.",
+          name: "Celeste",
+          imageLink:
+              "https://upload.wikimedia.org/wikipedia/commons/0/0f/Celeste_box_art_full.png"),
+      Game(
+          id: 5,
+          description:
+              "Baldur's Gate 3, Larian Studios tarafından geliştirilen ve Wizards of the Coast tarafından yayınlanan bir rol yapma video oyunudur. Oyun, Dungeons & Dragons 5th edition kurallarına dayanmaktadır. Oyun, 6 Ekim 2020'de Microsoft Windows ve Google Stadia için erken erişimde piyasaya sürüldü.",
+          name: "Baldur's Gate 3",
+          imageLink:
+              "https://image.api.playstation.com/vulcan/ap/rnd/202302/2321/ba706e54d68d10a0eb6ab7c36cdad9178c58b7fb7bb03d28.png"),
+      Game(
+          id: 6,
+          description:
+              "Ori and the Will of the Wisps, Moon Studios tarafından geliştirilen ve Xbox Game Studios tarafından yayınlanan bir platform oyunudur. Oyun, 11 Mart 2020'de Microsoft Windows ve Xbox One için piyasaya sürüldü. Oyun, 17 Eylül 2020'de Nintendo Switch için piyasaya sürüldü.",
+          name: "Ori and The Will of The Wispssss",
+          imageLink:
+              "https://upload.wikimedia.org/wikipedia/en/9/94/Ori_and_the_Will_of_the_Wisps.jpg"),
+    ];
+  }
+
+  changeMode() {
+    if (countValue == 2) {
+      setState(() {
+        countValue = 1;
+        aspectWidth = 3;
+        aspectHeight = 1;
+      });
+    } else {
+      setState(() {
+        countValue = 2;
+        aspectWidth = 2;
+        aspectHeight = 1;
+      });
+    }
+  }
+
+  getGridViewSelectedItem(BuildContext context, Game gridItem) {
+    showDialog(
+      context: context,
+      builder: (BuildContext context) {
+        return AlertWidget(
+          title: gridItem.name,
+          content: gridItem.description,
+        );
+      },
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        body: Column(children: [
+      Container(
+          margin: const EdgeInsets.fromLTRB(10, 10, 10, 10),
+          child: Button(
+            onPressed: () => changeMode(),
+            label: 'Change GridView Mode To ListView ',
```

Comment:
label seems static in wiki page. Should be better if you convert it to dynamic in future.